### ISSUE TITLE
Macos verion?

### ISSUE THREAD

**Issue by Flaming-Gamer101** [iAmAsval/FModel#246]
### Description

would it be possible to make a macOS version of model I would be willing to help with the programming of it but I want to know if it is possible

**Comment by NotOfficer** [comment]
Possible but will not (99%) be a thing.

**Comment by Flaming-Gamer101** [comment]
Ok, I will start seeing if can attempt to compile it

**Comment by NotOfficer** [comment]
You will have to recode the UI and other windows specific code

**Comment by Flaming-Gamer101** [comment]
ok thanks for telling me what i need to do

**Comment by NotOfficer** [comment]
I told you that you will fail

**Comment by Flaming-Gamer101** [comment]
no, you told me that I might fail but I don't care I'm going to try
![haters-haters-gonna-hate](https://user-images.githubusercontent.com/79016368/149535221-e74fb8ee-de8c-4d12-bcb3-8265b9d3aa46.gif)

**Comment by NotOfficer** [comment]
![bonk](https://cdn.discordapp.com/emojis/803389559445520414.gif?size=256&quality=lossless)

**Comment by GhostScissors** [comment]
What's the point any ways Fortnite is not available on apple anyways

**Comment by Flaming-Gamer101** [comment]
not for Fortnite this app has more uses that that

**Comment by Jawschamp** [comment]
> no, you told me that I might fail but I don't care I'm going to try
> 
> ![haters-haters-gonna-hate](https://user-images.githubusercontent.com/79016368/149535221-e74fb8ee-de8c-4d12-bcb3-8265b9d3aa46.gif)
> 
> 
> 
> 

Don't worry, he's just a dick, although I don't think that you'll be successful either, I think you can and should try to learn from it; and who knows, maybe you prove dick head wrong, GL!

**Comment by NotOfficer** [comment]
> If only .NET Core supported GUI on macOS and Linux, just like good old JavaFX. .NET Core is still useless as of today

\* .NET Core is outdated as of today

https://github.com/dotnet/maui

**Comment by xploree** [comment]
> * .NET Core is outdated as of today

you tell em

**Comment by tyl4r** [comment]
https://avaloniaui.net/XPF
